examples/src/main/scala/io/laserdisc/kafka/DangerousGDPR1.scala

```diff
@@ -0,0 +1,56 @@
+package io.laserdisc.kafka
```

Comment:
they should probably be moved under
```
examples/src/[main>test]/scala/...
```

Reply:
?? 🤔

Reply:
if you run `sbt test` these specs are ignored, but it's just an example, so no big deal